modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -152,13 +158,14 @@ NOTE: At minimum, `identityProvider_id` and `username` are required for `sub`, a
 | The authentication server. For example, the authentication server for the US region is `https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token`.
 
 |===
-+
 . The app calls the authorization server with the generated assertion.
 
 === Client Credentials
 
-The `client_credentials` grant type is suitable for machine-to-machine authentication where a specific user’s permission to access data is not required.
-It uses the `clientId` and `clientSecret` of the client to authorize and access protected data.
+Use the `client_credentials` grant type for machine-to-machine authentication where a specific user’s permission to access data isn't required.
+
+The `client_credentials` grant type uses the `clientId` and `clientSecret` of the client to authorize and access protected data.
+
 CI/CD pipelines using service accounts to programmatically call Anypoint Platform can be configured to use connected apps to prevent service disruptions when configuring xref:multi-factor-authentication.adoc[multi-factor authentication].
```

Comment:
It feels like this belongs with the "Create a Connected App That Acts on Its Own Behalf". Although I can see how it technically belongs under this "Grant Types and Token Flow" section as well. Maybe just add a link between the sections?